tools/tcpconnect_example.txt

```diff
@@ -41,16 +41,20 @@ process to various other addresses. A few connections occur every minute.
 USAGE message:
 
 # ./tcpconnect -h
-usage: tcpconnect [-h] [-t] [-p PID]
+usage: tcpconnect [-h] [-t] [-p PID] [-P PORT [PORT ...]]
 
 Trace TCP connects
 
 optional arguments:
   -h, --help         show this help message and exit
   -t, --timestamp    include timestamp on output
   -p PID, --pid PID  trace this PID only
+  -P PORT [PORT ...], --port PORT [PORT ...]
+                     trace these ports only
 
 examples:
     ./tcpconnect           # trace all TCP connect()s
     ./tcpconnect -t        # include timestamps
     ./tcpconnect -p 181    # only trace PID 181
+    ./tcpconnect -P 80     # only trace port 80
+    ./tcpconnect -P 80 81  # only trace port 80 and 81
```

Comment:
This is the default behaviour of argparse with `nargs='+'` from line 43 and comes free with it: https://docs.python.org/3/library/argparse.html#nargs 
I can make -P an optional string parameter and handle it by splitting around space of parameter. Whichever is preferred.